pages/docs/reference/sealed-classes.md

```diff
@@ -48,3 +93,19 @@ fun eval(expr: Expr): Double = when(expr) {
 }
 ```
 </div>
+
+## Try sealed interfaces and additional location for direct subclasses
```

Comment:
`Try additional location` seems unnatural. Maybe something like `Try placing direct subclasses in the same package` or `using direct subclasses from the same package`.